src/journal2ebook/_config.py

```diff
@@ -47,44 +52,56 @@ class Profile:
     def __str__(self):
         return self.name
 
-
 CONFIG_DEFAULT = {
     "last_dir": Path("~"),
     "last_profile": 0,
     "profiles": [Profile("Default")],
     "k2pdfopt_path": None,
 }
 
-
 class Config:
     def __init__(self):
         self._path = CONFIG_PATH
-        self._config = CONFIG_DEFAULT
+        self._config = CONFIG_DEFAULT.copy()  # Use a copy to avoid modifying the original
+
+        print(f"Config file should be located at: {self._path}")
```

Comment:
Suggest using the python logging library instead of print statements. This is best practice because it allows the end user to have more control of the logging level.